src/Traits/Categories/DeliveryHelpers.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace NotificationChannels\OneSignal\Traits\Categories;
+
+trait DeliveryHelpers
+{
+    /**
+     * Set the send after.
+     *
+     * @param string $date
+     *
+     * @return $this
+     */
+    public function setSendAfter(string $date)
+    {
+        return $this->setParameter('send_after', $date);
+    }
+
+    /**
+     * Set the deplayed option.
+     *
+     * @param string $delayedOption
+     *
+     * @return $this
+     */
+    public function setDelayedOption(string $delayedOption)
+    {
+        return $this->setParameter('delayed_option', $delayedOption);
+    }
+
+    /**
+     * Set the delivery at time of the day. Use with delayed option = timezone.
+     *
+     * @param string $timeOfDay
+     *
+     * @return $this
+     */
+    public function setDeliveryTimeOfDay(string $timeOfDay)
+    {
+        return $this->setParameter('delivery_time_of_day', $timeOfDay);
+    }
+
+    /**
+     * Set the Time to Live in Seconds.
+     *
+     * @param int $ttl
+     *
+     * @return $this
+     */
+    public function setTTL(int $ttl)
```

Comment:
As we are going with `Ios` of `iOS`, might want to do `Ttl`. I hate this capitalisation stuff with acronyms like that i.e `Html` etc. always feels weird.

Reply:
Yeah you are right :)